source/coll_db.f90

```diff
@@ -586,6 +586,16 @@ subroutine cdb_writeDB_newFromOld
     return
   end if
 
+  do j=1,cdb_nColl
+    ! Fixing family setting to that of the first collimator in the family when writing old CollDB in new format
+    ! Allows to take into account if settings were defined in fort.3 or old CollDB
+    if(j > 1 .and. cdb_cFamily(j) == cdb_cFamily(j-1)) then
+      continue
```

Comment:
Is `continue` a placeholder for coming code, or do you want it to continue the loop like in Python? If it's the latter, then it will not work as you intended, although since there is no more code in the do loop (at the moment) it will anyway. However, this can be a stumbling block in the future if code is added after the if-statement.

`continue` in Fortran is the same as `pass` in Python. `cycle` is the Fortran equivalent of Python's `continue`.

Reply:
Thanks for the remark, I fixed it in the latest commit.